entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "${DATABASE_URL}" ]; then
+    echo "DATABASE_URL variable not defined."
+    exit 1
+fi
+
+if [ -z "${REDIS_URL}" ]; then
+    echo "REDIS_URL variable not defined."
+    exit 1
+fi
+
+if [ -z "${SECRET_KEY_BASE}" ]; then
+    echo "SECRET_KEY_BASE variable not defined."
+    exit 1
+fi
+
+# Create database, if necessary
+bundle exec rake railties:install:migrations db:create db:migrate
```

Comment:
This doesn't work on an existing database. As `db:create` fails due the database already existing. I think you need something like the following:
```
bundle exec rake railties:install:migrations 2>/dev/null || bundle exec rake db:setup
```
i.e. Try to run the migrations, if it succeeds then the database and schema existed. If not, then run `db:setup`.

Reply:
Here is my interpretation of the rake tasks:
* db:migrate runs (single) migrations that have not run yet
* db:create creates the database
* db:drop deletes the database
* db:schema:load creates tables and columns within the (existing) database following schema.rb
* db:setup does db:create, db:schema:load, db:seed
* db:reset does db:drop, db:setup

That is why I suggest `db:setup`.

Reply:
`db:create` returns a 0 exit code if the DB already exists.
```
/usr/src/shipit # bundle exec rake db:create
D, [2019-04-04T15:05:01.004333 #47] DEBUG -- :    (0.6ms)  CREATE DATABASE "shipit_production" ENCODING = 'unicode'
Database 'shipit_production' already exists
/usr/src/shipit # echo $?
0
```